### DISCUSSION TITLE
Regenerate changelog from start

### DISCUSSION THREAD

**Discussion by borisdamevin** [semantic-release/semantic-release#2368] [Q&A]
Hi,

I would like to know if is possible to regenerate from beggining the CHANGELOG for have completed, clear and updated CHANGELOG.md, for example if we move to "semantic-release" under development?

Thank you.

**Comment by travi** [comment]
semantic-release does not provide the ability to regenerate from the beginning. it operates per release and can update your changelog  with each of those releases, but there is no ability to start over

**Reply by borisdamevin** [reply]
Thank for anwser.

Why not consider to provide this feature for project who integrate "semantic-release" under development?

Specialy if the development started from long time and dev team want cool new tools and have coherence dans le changelogs de maniére automatique.

**Comment by trallnag** [comment]
From what I understand `@semantic-release/release-notes-generator` uses `conventional-changelog` under the hood. `conventional-changelog`. So it should be possible to use `conventional-changelog` to regenerate the full changelog from the start?

I'm currently trying to achieve this with `conventional-changelog-cli`, but I'm only partially successful. No idea how to tell the cli to include more commit types in the changelog beyond features and bug fixes.

See here: https://github.com/conventional-changelog/conventional-changelog/issues/927

---

Same question: https://github.com/semantic-release/semantic-release/discussions/1983
